### ISSUE TITLE
How to add the others components from UI Fabric Core?

### ISSUE THREAD

**Issue by biapar** [johannes-z/office-ui-fabric-vue#8]
How to add the others components from UI Fabric Core like https://developer.microsoft.com/en-us/fabric#/components/documentcard?
Is't possible to use them into VUE without VUE components but direct from Fabric UI?

**Comment by johannes-z** [comment]
No, there is no out-of-the-box solution for this. Al lthe Vue components are rewritten in Vue, so if they don't exist, you can't use them.

There is however this package: https://github.com/akxcv/vuera
This has the downside though, that both `Vue` and `React` will have to be loaded.

**Comment by biapar** [comment]
Hi, this could be a track path https://aravindvoggu.in/uikit-with-vuejs-vue-cli-3-db811e43c46b
